### PR TITLE
Add a transparent background to the send reply button.

### PR DESCRIPTION
**Pull request by camilasan**
Before:
before-gray-bkg

After:
after-no-gray-bkg

### CODE REVIEW

**Review by claucambra — APPROVED** [review]
Approved but small comment

File: src/gui/tray/TalkReplyTextField.qml [hunk 1]
```diff
@@ -49,6 +49,9 @@ Item {
             flat: true
             enabled: replyMessageTextField.text !== ""
             onClicked: root.sendReplyMessage()
+            background: Rectangle {
+                color: "transparent"
+            }
```
[the remark this answers is not in the document]
Comment (camilasan) [reply src/gui/tray/TalkReplyTextField.qml:54]: > Is the background invisible when you set this property to `null`, `undefined`, or `Item {}` ? That way we don't have to instantiate a rectangle at all

Yes, it is invisible when set to null. Should I use null then?

[the remark this answers is not in the document]
Comment (claucambra) [reply src/gui/tray/TalkReplyTextField.qml:54]: I would say so, yeah, it is (ever so slightly) more efficient

**Review by mgallien — APPROVED** [review]

**Pull request merged**